x/upgrade/module.go

```diff
@@ -163,9 +163,6 @@ func (am AppModule) BeginBlock(ctx context.Context) error {
 	return BeginBlocker(ctx, am.keeper)
 }
 
-// IsUpgradeModule implements the module.UpgradeModule interface.
-func (am AppModuleBasic) IsUpgradeModule() {}
```

Comment:
does this need to be removed in 0.50 as well?

Reply:
This already has been reverted prior to tagging rc1 in v0.50